portals/admin/src/main/webapp/source/src/app/data/api.js

```diff
@@ -445,6 +456,18 @@ class API extends Resource {
         });
     }
 
+     /**
+     * Update an api's owner
+     */
+     updateApiProvider(apiId, provider) {
+        return this.client.then((client) => {
+            return client.apis["Api Provider Change"].providerNamePost(
```

Comment:
Change to single quotes instead of double. And make sure to build this with linting

Reply:
Fixed in https://github.com/wso2/apim-apps/pull/519/commits/29607f7f40ef583590bb286f3e1ea8d36af5091c